docs/buttons.md

```diff
@@ -57,12 +57,11 @@ _Mixin which helps to create button with specified border and background color w
 - `$active-border` - defines button's border when active (**default `$color`**)
 
 ### Usage: 
-Assigned button with $color-primary(variable) color and $color-primary-light(variable) color of border and background
- while active.
+Assigned button with `$color-red` color and `$color-red-dark` color of border while active.
```

Comment:
Assign `$color-red` as button's text and border color and `$color-red-dark` as border color while active.